testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jaxrs/tracing/TracingFeatureDefaultModeTestCase.java

```diff
@@ -0,0 +1,74 @@
+package org.jboss.as.test.integration.jaxrs.tracing;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.as.test.integration.jaxrs.packaging.war.WebXml;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.client.ClientBuilder;
+import java.net.URL;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class TracingFeatureDefaultModeTestCase {
+    @Deployment
+    public static Archive<?> deploy() {
+        WebArchive war = ShrinkWrap.create(WebArchive.class, "tracing.war");
+        war.addClasses(TracingConfigResource.class, TracingApp.class);
+        war.addAsWebInfResource(WebXml.get(
+                "    <context-param>\n" +
+                        "        <param-name>resteasy.server.tracing.type</param-name>\n" +
+                        "        <param-value>ALL</param-value>\n" +
+                        "    </context-param>\n" +
+                        "    <context-param>\n" +
+                        "        <param-name>resteasy.server.tracing.threshold</param-name>\n" +
+                        "        <param-value>VERBOSE</param-value>\n" +
+                        "    </context-param>\n" +
+                        "    <servlet>\n" +
+                        "        <servlet-name>resteasy-dispatcher</servlet-name>\n" +
+                        "        <servlet-class>org.jboss.resteasy.plugins.server.servlet.HttpServletDispatcher</servlet-class>\n" +
+                        "        <init-param>\n" +
+                        "            <param-name>jakarta.ws.rs.Application</param-name>\n" +
+                        "            <param-value>org.jboss.as.test.integration.jaxrs.tracing.TracingApp</param-value>\n" +
+                        "        </init-param>\n" +
+                        "    </servlet>\n" +
+                        "    <servlet-mapping>\n" +
+                        "        <servlet-name>resteasy-dispatcher</servlet-name>\n" +
+                        "        <url-pattern>/*</url-pattern>\n" +
+                        "    </servlet-mapping>"), "web.xml");
```

Comment:
This shouldn't be required with the `TracingApp`.

Reply:
okay!